src/extension.ts

```diff
@@ -172,7 +172,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         findTreeItem,
         pickTreeItem,
         revealTreeItem,
-        apiVersion: '1.1.0'
+        apiVersion: '2.0.0'
```

Comment:
I think you can still do 1.2.0. The only stuff you changed was interface and type names, but those disappear when typescript gets compiled to javascript anyways.